docs/docusaurus/docs/guides/setup/configuring_data_contexts/how_to_convert_an_ephemeral_data_context_to_a_filesystem_data_context.md

```diff
@@ -0,0 +1,109 @@
+import ConnectingToDataFluently from '/docs/components/connect_to_data/link_lists/_connecting_to_data_fluently.md'
+import SetupConfigurations from '/docs/components/setup/link_lists/_setup_configurations.md'
+
+## Introduction
+
+An Ephemeral Data Context is a temporary, in-memory Data Context that will not persist beyond the current Python session.  However, if you decide you would like to save the contents of an Ephemeral Data Context for future use you can do so by converting it to a Filesystem Data Context.
+
+## Prerequisites
+
+<Prerequisites>
+
+- A working installation of Great Expectations
+- An Ephemeral Data Context instance
+- A passion for Data Quality
+
+</Prerequisites> 
+
+<details>
+<summary>
+
+### If you still need to set up and install GX...
+
+</summary>
+
+<IfYouStillNeedToSetupGx />
+
+</details>
+
+
+<details>
+<summary>
+
+### If you still need to create a Data Context...
+
+</summary>
+
+The `get_context()` method will return an Ephemeral Data Context if your system is not set up to work with GX Cloud and a Filesystem Data Context cannot be found.  For more information, see:
+- [How to quickly instantiate a Data Context](/docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_quickly_instantiate_a_data_context.md)
+
+You can also explicitly instantiate an Ephemeral Data Context (for those occasions when your system is set up to work with GX Cloud or you do have a previously initialized Filesystem Data Context).  For more information, see:
+- [How to explicitly instantiate an Ephemeral Data Context](/docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_explicitly_instantiate_an_ephemeral_data_context.md)
+
+</details>
+
+<details>
+
+<summary>
+
+### If you aren't certain that your Data Context is Ephemeral...
+
+</summary>
+
+You can easily check to see if you are working with an Ephemeral Data Context with the following code (in this example, we are assuming your Data Context is stored in the variable `context`):
+
+```python title="Python code"
+from great_expectations.data_context import EphemeralDataContext
+
+# ...
+
+if isinstance(context, EphemeralDataContext):
+    print(It's Ephemeral!)
+```
+
+</details>
+
+## Steps
+
+### 1. Verify that your current working directory does not already contain a GX Filesystem Data Context
+
+The method for converting an Ephemeral Data Context to a Filesystem Data Context initializes the new Filesystem Data Context in the current working directory of the Python process that is being executed.  If a Filesystem Data Context already exists at that location, the process will fail.
+
+You can determine if your current working directory already has a Filesystem Data Context by looking for a `great_expectations.yml` file.  The presence of that file indicates that a Filesystem Data Context has already been initialized in the corresponding directory.
```

Comment:
I get that we want a separate doc on how to quickly instantiate a Data Context, but it feels odd to have this doc which is essentially a single line of code to run. I also know that I specifically requested this doc as a bridge from Quickstart.

One proposal would be to have a doc called something like "How to get and save a Data Context quickly" which has two steps, one for getting and one for saving. I don't love that either, tbh, but having the user need to click to an extra doc to run one line of code doesn't sit right with me. I'm open to brainstorming here.

Reply:
I'm making an edit that adds an abbreviated explanation of the `convert_to_file_context()` method to an admonition box that can be included in both "How to quickly get a Data Context" and "How to explicitly instantiate an Ephemeral Data Context".  The admonition box directs people to "How to convert an Ephemeral Data Context to a Filesystem Data Context" if they are interested in a more detailed explanation of the `convert_to_file_context()` method.

I think this might be the best option -- I don't want to add steps to "How to quickly get a Data Context" that would essentially be flagged as "optional, only if you are using an Ephemeral Data Context" when we can assume that a fair number of people won't be using an Ephemeral Data Context.

One guide = one process, so the idea is to minimize the explicitly optional steps while still including the information that may be relevant.